java-checks-test-sources/src/main/java/checks/PseudoRandomCheck.java

```diff
@@ -26,7 +30,7 @@ void fun() {
     double abs = Math.abs(12); // Compliant
 
     // java.util.concurrent.ThreadLocalRandom
-    int rand2 = ThreadLocalRandom.current().nextInt();  // Noncompliant
+    int rand2 = ThreadLocalRandom.current().nextInt();  // Noncompliant [[sc=35;ec=42]]
```

Comment:
Here the issue location changed from the call to `nextInt` to the call to `current`. I think this is fine and is anyway more consistent with how we handle `java.util.Random` (where we would also raise on the constructor call, not the call to `nextInt`).